From 32d57440140b70ab4019f17ae6e8f63420a40cfd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 10 Sep 2019 15:51:38 +0200
Subject: [PATCH 036/152] build: Exclude private headers from GIR

They contain API that is explicitly not meant to be used externally
and - as it uses the "wrong" namespace due to the _ prefix - doesn't
end up in the introspection data anyway.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/709
---
 src/st/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/st/meson.build b/src/st/meson.build
index 4cbebace7..45cc5c3c0 100644
--- a/src/st/meson.build
+++ b/src/st/meson.build
@@ -93,7 +93,7 @@ st_enums = gnome.mkenums_simple('st-enum-types',
 #endif'''
 )
 
-st_gir_sources = st_sources + st_private_headers + st_headers + st_enums
+st_gir_sources = st_sources + st_headers + st_enums
 
 data_to_c = find_program(meson.source_root() + '/src/data-to-c.pl')
 
-- 
2.23.0

